### ISSUE TITLE
The chat bubbles should not have an accessibility label

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1326]
### Description of problem

iOS is exposing an accessibility label for a offer chat bubble but it is not something that can be interacted with.

### Expected behavior

Accessibility should be off for this chat bubble.

### Steps to reproduce

1. With voice control on;
2. Receive and offer or profo request;
3. Say "Show names";
4. Say "Touch" ... along with whatever the accessibility label from 3 is.

### Screenshots and/or log output

![IMG_0068.PNG](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/65c3cc68-808b-4e03-984b-bf076b518dc7)

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by bryce-mcmath** [comment]
To be more specific the `? Best BC College sent a credential offer` (and other chat messages) should not have an a11y label, as they cannot be interacted with. The `View offer` a11y label is good, and should be kept.

**Comment by jleach** [comment]
Fixed. Tested in 1.0.22 (2138)

![Image](https://github.com/user-attachments/assets/abc3cdc3-5872-4eaf-aa30-b1d07f54f3a8)
